urllib3/response.py

```diff
@@ -1,9 +1,14 @@
+try:
+    import http.client as httplib
+except ImportError:
+    import httplib
 import zlib
 import io
 from socket import timeout as SocketTimeout
 
 from ._collections import HTTPHeaderDict
-from .exceptions import ProtocolError, DecodeError, ReadTimeoutError
+from .exceptions import ProtocolError, DecodeError, ReadTimeoutError, \
+    ResponseNotChunked
```

Comment:
Rather than using the explicit line continuation, prefer using python's implicit line continuation between parentheses:

```python
from .exceptions import (
    ProtocolError, DecodeError, ReadTimeoutError, ResponseNotChunked
)
```